Add script to manage solver authentication (#282)
gnosis/gp-v2-contracts: README.md

This package additonally contains a `networks.json` file at the root with the
address of each deployed contract as well the hash of the Ethereum transaction
used to create the contract.

> ## Solver authentication

Reviewer: ```suggestion
## Solver Authentication
```

> This repo defines scripts to manage the list of authenticated solver in all
> networks the contract is deployed to.

Reviewer: ```suggestion
This repo contains scripts to manage the list of authenticated solvers in all networks the contract is deployed to.
```

The scripts are called with:

```
yarn solvers command [arg ...]
```

Here is a list of available commands.

> The commands flagged with [*] require the private key of the authentication
> contract owner to be available to the script, for example by exporting it with

Reviewer: Just a markdown nit. Each sentence should remain on a single line.
```suggestion
The commands flagged with [*] require the private key of the authentication contract owner to be available to the script, for example by exporting it with
```
